components/crud-web-apps/common/frontend/kubeflow-common-lib/README.md

```diff
@@ -1,27 +1,43 @@
-# AngularFrontendKubeflow
-
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.20.
-
-## Development server
-
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
-
-## Code scaffolding
-
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
-
-## Build
-
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+# Kubeflow Common Frontend Library
+
+This code provides a common library of reusable Angular Components that can be used from our different Kubeflow web apps. This library aims to:
+* Enforce a common UX throughout the different apps
+* Reduce the development effort required to propagate changes to all the web apps
+* Minimize the code duplication between our Kubeflow web apps
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.20, which is required to build and run the unit tests.
+
+## Local development
+In order to use this library while developing locally  an Angular app you will need to:
+1. Build the `kubeflow` node module from this source code
+2. Link the produced module to your global npm modules
+3. Link the `kubeflow` module in the npm modules of you app
```

Comment:
nit: your

Reply:
Good catch, thanks!
Pushed a new commit that fixes if